app/index.html

```diff
@@ -17,6 +17,9 @@
   <link rel="stylesheet" href="css/simplescrollbars.css">
   <link rel="stylesheet" href="css/base16-tomorrow-light.css">
   <link rel="stylesheet" href="css/base16-ocean-dark.css">
+  <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/vex-js/3.0.0/css/vex.min.css" /> -->

```

Comment:
We don't need a CDN now that we're using styles from the package itself (line 21, and 22), right?

Reply:
Oh, apologies, it's just a comment. 👍